test/verify/check-metrics

```diff
@@ -626,6 +626,15 @@ class TestMultiCPU(MachineCase):
         self.assertGreaterEqual(getMaximumSpike(b, "cpu", False, 1598968800000, 45), 0.5)
         self.assertLessEqual(getMaximumSpike(b, "cpu", False, 1598968800000, 45), 1.0)
 
+        # Test current usage of cores
+        b.wait(lambda: progressValue(b, "#current-cpu-usage") < 20)
+        m.execute("systemd-run --collect --slice cockpittest -p CPUQuota=60% --unit cpu-hog dd if=/dev/urandom of=/dev/null")
+        m.execute("systemd-run --collect --slice cockpittest -p CPUQuota=30% --unit cpu-piglet dd if=/dev/urandom of=/dev/null")
+        b.wait(lambda: progressValue(b, "#current-cpu-usage") > 40)
+        b.mouse("#current-cpu-usage", "mouseenter")
+        b.wait(lambda: int(b.text(".pf-c-tooltip .pf-l-flex:first-child div:nth-child(2)")[:-1]) > 50)
+        b.wait(lambda: int(b.text(".pf-c-tooltip .pf-l-flex:first-child div:nth-child(2)")[:-1]) > 20)
```

Comment:
... and perhaps that the "total" CPU usage is > 40 and < 50? (average between the two cores)

Reply:
I had `> 40` and I added `< 50` but the first run had some with `51%`. This smells like it might make this test flaky, but lets see.

Reply:
nah, now the machine was somehow busy and it was 94%. I'll drop it.

Reply:
Right, that was a thinko -- the CPU is probably busy with stuff other than these two synthetic CPU eater services.